src/sidebar/components/test/selection-tabs-test.js

```diff
@@ -1,245 +1,249 @@
-      assert.include(tabs[0].textContent, '123');
-      assert.include(tabs[1].textContent, '456');
+      const wrapper = createDeepComponent();
+      const tabs = wrapper.find('a');
+      assert.isTrue(tabs.at(0).contains('Annotations'));
+      assert.equal(
+        tabs
+          .at(0)
+          .find('.selection-tabs__count')
+          .text(),
+        123
+      );
+      assert.isTrue(tabs.at(1).contains('Page Notes'));
+      assert.equal(
+        tabs
+          .at(1)
+          .find('.selection-tabs__count')
+          .text(),
+        456
+      );
     });
 
     it('should display annotations tab as selected', function() {
-      const elem = util.createDirective(document, 'selectionTabs', {
-        selectedTab: 'annotation',
-        totalAnnotations: '123',
-        totalNotes: '456',
-      });
-      const tabs = elem[0].querySelectorAll('a');
-
-      assert.isTrue(tabs[0].classList.contains('is-selected'));
+      const wrapper = createDeepComponent();
+      const aTags = wrapper.find('a');
+      assert.isTrue(aTags.at(0).hasClass('is-selected'));
     });
 
     it('should display notes tab as selected', function() {
-      const elem = util.createDirective(document, 'selectionTabs', {
-        selectedTab: 'note',
-        totalAnnotations: '123',
-        totalNotes: '456',
+      const wrapper = createDeepComponent({
+        selectedTab: uiConstants.TAB_NOTES,
       });
-      const tabs = elem[0].querySelectorAll('a');
-
-      assert.isTrue(tabs[1].classList.contains('is-selected'));
+      const tabs = wrapper.find('a');
+      assert.isTrue(tabs.at(1).hasClass('is-selected'));
     });
 
-    it('should not show the clean theme when settings does not contain the clean theme option', function() {
-      const elem = util.createDirective(document, 'selectionTabs', {
-        selectedTab: 'annotation',
-        totalAnnotations: '123',
-        totalNotes: '456',
+    it('should display orphans tab as selected if there is 1 or more orphans', function() {
+      const wrapper = createDeepComponent({
+        selectedTab: uiConstants.TAB_ORPHANS,
+        totalOrphans: 1,
       });
-
-      assert.isFalse(
-        elem[0]
-          .querySelectorAll('.selection-tabs')[0]
-          .classList.contains('selection-tabs--theme-clean')
-      );
+      const tabs = wrapper.find('a');
+      assert.isTrue(tabs.at(2).hasClass('is-selected'));
     });
 
-    it('should show the clean theme when settings contains the clean theme option', function() {
-      angular.mock.module('app', {
-        store: {},
-        features: {
-          flagEnabled: sinon.stub().returns(true),
-        },
-        settings: { theme: 'clean' },
-      });
-
-      const elem = util.createDirective(document, 'selectionTabs', {
-        selectedTab: 'annotation',
-        totalAnnotations: '123',
-        totalNotes: '456',
+    it('should not display orphans tab as if there is 0 orphans', function() {
```

Comment:
```suggestion
    it('should not display orphans tab if there are 0 orphans', function() {
```

Pardon the pedantry 😛